README.markdown

```diff
@@ -194,19 +194,13 @@ Follow the instructions below if you are updating from Symphony 2.0 (not from Gi
 
 Symphony uses [Grunt](http://gruntjs.com/) to build concatenated and minified assets:
 
-1. Install `grunt-cli` globally ([see the Grunt documentation](http://gruntjs.com/getting-started)):
-
-		npm install -g grunt-cli
```

Comment:
If we remove this, don't we need to add the `grunt-cli` package into package.json ? I think the idea behind having the cli vs the lib is that it allows you to have a specific grunt version per project, but keep a single cli tool around that calls in the right version. From http://gruntjs.com/getting-started: 

> Note that installing grunt-cli does not install the Grunt task runner! The job of the Grunt CLI is simple: run the version of Grunt which has been installed next to a Gruntfile. This allows multiple versions of Grunt to be installed on the same machine simultaneously.